### DISCUSSION TITLE
Kp-index

### DISCUSSION THREAD

**Discussion by jacerny** [tcarwash/home-assistant_noaa-space-weather#46] [Ideas]
is possible to add Kp-index?
https://www.spaceweatherlive.com/cs/auroralni-aktivita/kp-index.html

**Comment by tcarwash** [comment]
This should be what's labeled at planetary k index, which may be a misunderstanding on my part. If so let me know!

**Comment by jacerny** [comment]
I am not sure. The data are not consistent over time and give distorted information throughout the day. The recorded data is dropping to zero, but that's not true... Please, se you the attachement.
![KP-index](https://github.com/tcarwash/home-assistant_noaa-space-weather/assets/89773333/63345e00-be5b-489d-9910-709901bd79eb)

**Reply by tcarwash** [reply]
I see what you're saying in the attachment, it shouldn't be returning 0 between measurements, it just should be the same as the previous measurement. I'll have to dig deeper into the code. I think I have my grafana stuff filling with the previous measurement which has probably masked this issue for me personally
